srcpkgs/Terminal.app/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'Terminal.app'
+pkgname=Terminal.app
+version=0.9.9
+revision=1
+wrksrc="Terminal-${version}"
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui"
```

Comment:
When I attempt to launch this, I get
```
020-12-17 13:33:08.904 Terminal[11578:11578] Did not find correct version of backend (libgnustep-back-028.bundle), falling back to std (libgnustep-back.bundle).
2020-12-17 13:33:08.904 Terminal[11578:11578] NSApplication.m:304  Assertion failed in initialize_gnustep_backend.  Unable to find backend back
Terminal: Uncaught exception NSInternalInconsistencyException, reason: NSApplication.m:304  Assertion failed in initialize_gnustep_backend.  Unable to find backend back
```
Perhaps you are missing some more `depends`?

Reply:
`gnustep-back` most likely.

Reply:
Seems functional in my Void VM, should be fixed now.